client/core/localetest/main.go

```diff
@@ -2,32 +2,28 @@ package main
 
```

Comment:
Would this file do better in `/dcrdex/client/cmd/localetest`?

Reply:
I'm inclined to keep it here because it's not part of the product really, rather, a dev tool.